packages/flame/src/Dropdown/Dropdown.tsx

```diff
@@ -113,7 +113,6 @@ export const Dropdown: React.FC<Props> = ({
   zIndex = 1,
   children,
   onClick,
-  ...restProps
```

Comment:
Wouldn't that be considered breaking?

Reply:
gonna revert the changes done to this file. Was debugging some wack issues with storybook and usepopper and forgot to revert.